### ISSUE TITLE
[Feature request] Unread Auto-empty 

### ISSUE THREAD

**Issue by Neules** [normen/whatscli#25]
I'd like the program remove the Number of unread messages when I focus on that chat. I mean, whenever I focus on a chat this unread messsages are not unread anymore, but they remains besides the name of the contact.

**Comment by normen** [comment]
Yep, you have to manually reset the counter at the moment. Ctrl-n is the default mapping for that.

Adding such a feature is proably in order, I'll keep this open. PRs are welcome.

Am 02.12.2020 10:40 schrieb Neules:
